src/components/CohortCard.astro

```diff
@@ -0,0 +1,50 @@
+---
+const cardsList = [
+  {
+    name: "Learning Bitcoin From The Command Line",
+    img: "/website-v2/cohort/lbtcl.png",
+    no: "02",
+    desc: "Donec laoreet congue justo at rhoncus. Nam aliquet suscipit feugiat...",
+    bgColor: "bg-orange",
+    buttonBg: "bg-white",
+    buttonText: "text-black",
```

Comment:
These need not be part of config. We can take the index of card and if its odd, do bg-orange or if its even bg-black ?